src/controllers/chatTribes.ts

```diff
@@ -509,7 +509,7 @@ export async function receiveMemberApprove(payload) {
 
   const amount = chat.priceToJoin || 0
   const theChat = chat.dataValues || chat
-  const theOwner = owner.dataValues || owner
```

Comment:
So I'm not sure if this line is causing the tests to not pass but I'm not sure. This seems the most suspect though

Reply:
finally figured it out, it was an issue in app.ts. Should be good now, lets see if all the tests pass